webrtc-stats.html

```diff
@@ -2456,6 +2466,81 @@ <h2>
           </section>
         </div>
       </section>
+
+      <section id="playoutstats-dict*">
+        <h3>
+          {{RTCAudioPlayoutStats}} dictionary
+        </h3>
+        <p>
+          Only applicable if the playout path represents an audio devcice.
+          Represents one playout path, which may or may not be after mixing.
+        </p>
+        <div>
+          <pre class="idl">dictionary RTCAudioPlayoutStats : RTCStats {
+             double synthesizedSamplesDuration;
+             double totalSamplesDuration;
+             double totalPlayoutDelay;
+             double totalSamplesCount;
+};</pre>
+          <section>
+            <h2>
+              Dictionary {{RTCAudioPlayoutStats}} Members
+            </h2>
+            <dl data-link-for="RTCAudioPlayoutStats" data-dfn-for="RTCAudioPlayoutStats" class=
+            "dictionary-members">
+              <dt>
+                <dfn>synthesizedSamplesDuration</dfn> of type <span class=
+                "idlMemberType">double</span>
+              </dt>
+              <dd>
+                <p>
+                  If the playout path is unable to produce audio samples on time
+                  for device playout, samples are synthesized to be playout out
+                  instead. {{synthesizedSamplesDuration}} is incremented each
+                  time this happens. Can be used together with
+                  {{totalSamplesDuration}} to calculate the percentage of media
+                  played out being synthesized.
+                </p>
+              </dd>
+              <dt>
+                <dfn>totalSamplesDuration</dfn> of type <span class=
+                "idlMemberType">double</span>
+              </dt>
+              <dd>
+                <p>
+                  The total duration, in seconds, of all audio samples that have
+                  been playout.
+                </p>
+              </dd>
+              <dt>
+                <dfn>totalPlayoutDelay</dfn> of type <span class=
+                "idlMemberType">double</span>
+              </dt>
+              <dd>
+                <p>
+                  When audio samples are pulled by the playout device, this
+                  counter is incremented with the estimated delay of the playout
+                  path for that audio sample. Can be used together with
+                  {{totalSamplesCount}} to calculate the average playout delay
+                  per sample.
```

Comment:
Can you elaborate when the timer starts for this? I understand the timer ends when the device plays out the audio (for reals!) 

Does the timer start when it the audio device asks and gets the samples.